python/ray/worker.py

```diff
@@ -621,6 +621,7 @@ def init(
         _metrics_export_port: Optional[int] = None,
         _system_config: Optional[Dict[str, str]] = None,
         _tracing_startup_hook: Optional[Callable] = None,
+        _default_actor_lifetime: str =None,
```

Comment:
I think the default value should be specified in the job config. It should not be a per-node configuration. We don't want users to start two Ray nodes with different default lifetime values.

Reply:
Moved it to job config instead of `ray.init`